util/populate_dev_data.go

```diff
@@ -40,12 +40,17 @@ func main() {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 	flag.Parse()
 
-	ctx, err := getRemoteAPIContext()
-	if err != nil {
-		log.Fatal(err)
+	if *project != "" {
+		os.Setenv("DATASTORE_PROJECT_ID", *project)
```

Comment:
where is it being used?

Reply:
This and `DATASTORE_EMULATOR_HOST` are later used by the datastore client (in `NewClient`).

Reply:
oh right! I got confused for a moment